code/src/components/TodoList.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import Checkbox from "@material-ui/core/Checkbox";
+
+import todos from "../reducers/todos";
+import "./TodoListStyle.css";
+
+const TodoList = () => {
+  const dispatch = useDispatch();
+  const items = useSelector((store) => store.todos.items);
+
+  return (
+    <div>
```

Comment:
I realised I have just the same issue as you. When I write something long, it does not fit inside the task-section. It goes way beyond the intended width and messes up the layout. What we want is for the text to line-break.. Let me know if you have a good solution for that!!